datenguide_python/query_builder.py

```diff
@@ -335,13 +479,41 @@ def add_field(self, field: Union[str, Field], default_fields=True) -> Field:
         else:
             return self.start_field.add_field(field, default_fields=default_fields)
 
-    def get_graphql_query(self) -> str:
+    def drop_field(self, field: str) -> "Query":
+        if self.start_field.name == "allRegions":
+            if self.region_field is not None:
```

Comment:
Shouldn't this rather check, that the field being dropped is not the region_field? This way it only throws the error if after dropping the region field we try to drop another field.

Reply:
I dont think it is possible to drop the regions field the way its programmed now.
It should actually not be possible to reach the Error as a user, as the regions field is always created in the all_regions() function.

Reply:
Ok I understand. I would vote for a RuntimeError again in that case.